canonical-web-and-design/maas-ui
# fix(ui): redirect when machine deleted

## Done

- Redirect to the machine list when a machine is deleted.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to machine details.
- Use the take action menu to delete the machine.
- You should get redirected to the machine list.

## Fixes

Fixes: #1914.
